spec/features/user_signin_with_github_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature "user signs in with Github" do
+  scenario "is able to log in" do
+    mock_github_oauth
+
+    visit '/'
+    click_on('Login with Github')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.